lib/output.js

```diff
@@ -188,9 +188,10 @@ function withMetadata (options) {
  * @throws {Error} unsupported format or options
  */
 function toFormat (format, options) {
-  const actualFormat = formats.get(is.object(format) && is.string(format.id) ? format.id : format);
+  const formatLowerCase = format.toLowerCase()
```

Comment:
`format` might be an object so the call to `.toLowerCase()` needs to happen on the next line, something like the following (untested):
```diff
- ... formats.get(is.object(format) && is.string(format.id) ? format.id : format);
+ ... formats.get((is.object(format) && is.string(format.id) ? format.id : format).toLowerCase());
```